src/lib/pages/account-details/components/tables/transactions/TxsTableRow.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  Box,
+  Flex,
+  Grid,
+  Icon,
+  Tag,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { useEffect, useState } from "react";
+import { MdCheck, MdClose, MdKeyboardArrowDown } from "react-icons/md";
+
+import { RenderActionMessages } from "lib/components/action-msg/ActionMessages";
+import { ExplorerLink } from "lib/components/ExplorerLink";
+import { TableRow } from "lib/components/table";
+import { AccordionTx } from "lib/components/table/AccordionTx";
+import type { PastTransaction } from "lib/types";
+import { dateFromNow, formatUTC } from "lib/utils";
+
+interface TxsTableRowProps {
+  transaction: PastTransaction;
+  templateColumnsStyle: string;
+}
+
+export const TxsTableRow = ({
+  transaction,
+  templateColumnsStyle,
+}: TxsTableRowProps) => {
+  const { isOpen, onToggle } = useDisclosure();
+  const [isAccordion, setIsAccordion] = useState(false);
+  const [showCopyButton, setShowCopyButton] = useState(false);
+  useEffect(() => {
+    if (transaction.messages.length > 1) setIsAccordion(true);
+  }, [transaction.messages]);
```

Comment:
remove useState and useEffect, then just have
`const isAccordian = transaction.messages.length > 1`